src/ch3-Greedy/index.js

```diff
@@ -0,0 +1,34 @@
+const solution_02 = require("./02/solution");
+const solution_03 = require("./03/solution");
+const solution_04 = require("./04/solution");
+
+const solutions = {
+  getInputs: function (dir) {
+    return require("fs")
+      .readFileSync(require("path").resolve(__dirname, `${dir}/input.txt`))
+      .toString()
+      .split("\n");
+  },
+
+  solution_02: function () {
+    const input = this.getInputs("02");
+    const [n, m, k] = input[0].split(" ").map(Number);
+    const nums = input[1].split(" ").map(Number);
+    return solution_02(n, m, k, nums);
+  },
+
+  solution_03: function () {
+    const [, ...input2] = this.getInputs("03");
+    const arr = [...input2].map((v) => v.split(" ").map(Number));
+    return solution_03(arr);
+  },
+
+  solution_04: function () {
+    const [n, k] = this.getInputs("04")[0].split(" ").map(Number);
+    return solution_04(n, k);
+  }
+};
```

Comment:
```suggestion
const solutions = {
  getInputs(dir) {
    return require("fs")
      .readFileSync(require("path").resolve(__dirname, `${dir}/input.txt`))
      .toString()
      .split("\n");
  },

  solution_02() {
    const input = this.getInputs("02");
    const [n, m, k] = input[0].split(" ").map(Number);
    const nums = input[1].split(" ").map(Number);
    return solution_02(n, m, k, nums);
  },

  solution_03() {
    const [, ...input2] = this.getInputs("03");
    const arr = [...input2].map((v) => v.split(" ").map(Number));
    return solution_03(arr);
  },

  solution_04() {
    const [n, k] = this.getInputs("04")[0].split(" ").map(Number);
    return solution_04(n, k);
  }
};
```

요렇게 하면 더 깔끔하게 쓰실 수 있지 않을까 싶네요 :+1:

Reply:
@bigsaigon333   저도 동희님 의견에 공감합니다.  무심코 지나간 부분 이었는데 알려주셔서 감사합니다! 이런 활발한 리뷰 너무 좋네요! 👍